contracts/Comptroller.sol

```diff
@@ -1277,12 +1252,12 @@ contract Comptroller is ComptrollerV5Storage, ComptrollerInterface, ComptrollerE
      * @dev Note: If there is not enough COMP, we do not perform the transfer all.
      * @param user The address of the user to transfer COMP to
      * @param amount The amount of COMP to (possibly) transfer
-     * @return The amount of COMP which was NOT transferred to the user
+     * @return The amount of COMk which was NOT transferred to the user
```

Comment:
let's stick with calling it COMP here, not possible to change the token symbol at this point in time.